sbt-play-ebean/src/sbt-test/sbt-ebean/enhancement/build.sbt

```diff
@@ -1,6 +1,7 @@
 lazy val root = project
   .in(file("."))
   .enablePlugins(PlayJava, PlayEbean)
+  .settings(scalaVersion := "2.11.7")
```

Comment:
This doesn't need to be in `.settings`, since it's for the root project, it can just go with all the other settings on a line of its own.